veevavault/1.3/modules/ROOT/pages/veevavault-connector-config-topics.adoc

```diff
@@ -806,24 +222,24 @@ The following are the error codes that are caught in the Error Handling componen
 [[streaming_and_pagination]]
 == Streaming and Pagination
 
-All the operations in the connector (except Download Document) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
+All the operations in the connector (except *Download Document*) return an InputStream as a payload with respective results based on operation output. Because of this, by default, Mule applies Streaming Strategies. See xref:mule-runtime::streaming-about.adoc[Mule Streaming Strategies] for more details. The streaming strategies configuration fields are in the *Advanced* tab of the connector operations.
 
 image::veevavault-connector-streaming-pagination.png[image,width=601,height=124]
 
 The following operations in the connector provide a pagination mechanism based on Mule standard pagination.
 
-* <<get_documents,Get Documents>>
-* <<get_object_records,Get Object Records>>
-* <<query,Query>>
-* <<get_audit_details,Get Audit Details>>
+* *Get Documents*
+* *Get Object Records*
+* *Query*
+* *Get Audit Details*
 
-While using the paginated operations make sure to place a `For-Each/Splitter` element to retrieve each object (metadata is in JSON format) at a time. The pagination operations have *Fetch Size* and *Batch Size* fields.
+While using the paginated operations, make sure to place a `For-Each/Splitter` element to retrieve each object (metadata is in JSON format) at a time. The pagination operations have *Fetch Size* and *Batch Size* fields.
 
 * *Fetch Size* +
-The Fetch Size is a limit number of records that can be retrieved in a single page. The operation returns the pages with the fetch size number of JSON object records. +
+A limit number of records that can be retrieved in a single page. The operation returns the pages with the fetch size number of JSON object records. +
 In some cases, Veeva APIs auto-calculate the fetch size (number of records on each page) based on record size and the calculation exceeds the standard record size. The operation returns calculated records on each page.
 * *Batch Size* +
-The Batch Size is the number of pages to return in each batch and each page will have the fetch size number of records. The operation returns a number of records (metadata in JSON format) per batch, and is calculated like in the following example:
+The number of pages to return in each batch. Each page contains the fetch size number of records. The operation returns a number of records (metadata in JSON format) per batch, and is calculated like in the following example:
```

Comment:
Same questions: we are defining *Batch Size*. Do we need to explicitly relate this to Fetch Size? And further explicitly relate this to the number of records per batch: are we saying that pages=records?